src/menu.js

```diff
@@ -134,6 +140,7 @@ class Menu extends Component {
         <MenuStatus
           highlightedIndex={highlightedIndex}
           inputValue={inputValue}
+          getA11yStatusMessage={this.props.getA11yStatusMessage}
```

Comment:
maybe use the `getA11yStatusMessage` variable from the destructured props above?